### ISSUE TITLE
when could you upload full python code

### ISSUE THREAD

**Issue by YuhaoYeSteve** [SRITANU/SemanticSoftSegmentation#1]
I would very appreciate it

**Comment by SRITANU** [comment]
Yes, I have stopped it midway as you have to calculate eigenvalues of a huge sparse matrix, python is taking hours to calculate that. Matlab is doing the same in a few minutes. You can find my interaction with a former scipy contributor on Twitter : https://twitter.com/Combalgorythm/status/1129787645679218693 .

Also, if you want to have the Laplacian sparse .npz file it's here: https://drive.google.com/file/d/1RU0kH3rVf6TMILI4Z6Y0E-E_cFHFRrYo/view

Anyway, if you want to use the current technique for production, it's terribly slow. Lots of optimisation needs to be done in order to use the technique outlined in the paper. I am really busy with work till August. Then, I'll again have a look at how to optimize it. 

I am not closing this issue as it will keep me motivated to keep working on the problem.

**Comment by YuhaoYeSteve** [comment]
Thanks for your reply. Actually, I am trying to do a function like manually using photoshop to cut out an object in a picture and matting the cut area into a pure white picture, but automatically. I am new to this area and wondering if this paper is a good way to reach my goal. What I have already tried is using object detection(faster rcnn) algorithm to get the bounding box of the object and then use Poisson fusion to combine cut bounding box area and a pure white picture.
![image](https://user-images.githubusercontent.com/33405011/59566408-9753fe80-9092-11e9-94d6-55c1fb52bd28.png)

**Comment by YuhaoYeSteve** [comment]
The picture is the result of Faster RCNN and Poisson fusion. However, the final effect is not like what we see in the beautiful poster or TV advertisement.
